<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlissBeneficios</class>
 <widget class="QWidget" name="BlissBeneficios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tipo de Arquivo</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="campoImportacaoRubi">
        <property name="text">
         <string>Arquivo Importação Rubi (Senior)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="campoImportacaoVT">
        <property name="text">
         <string>Arquivo Importação SET(Empresa VT)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="campoLocalDoArquivo">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="botaoProcessar">
       <property name="text">
        <string>Processar</string>
       </property>
       <property name="icon">
        <iconset resource="thema.qrc">
         <normaloff>:/images/compass.png</normaloff>:/images/compass.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botaoExportar">
       <property name="text">
        <string>Exportar</string>
       </property>
       <property name="icon">
        <iconset resource="thema.qrc">
         <normaloff>:/images/txt-file-symbol.png</normaloff>:/images/txt-file-symbol.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>338</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>campoImportacaoRubi</tabstop>
  <tabstop>campoImportacaoVT</tabstop>
  <tabstop>campoLocalDoArquivo</tabstop>
  <tabstop>botaoProcessar</tabstop>
  <tabstop>botaoExportar</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources>
  <include location="thema.qrc"/>
 </resources>
 <connections/>
</ui>
